lib/internal/Magento/Framework/Stdlib/DateTime/Timezone.php

```diff
@@ -174,7 +183,17 @@ public function date($date = null, $locale = null, $useTimezone = true, $include
                     $timeType,
                     new \DateTimeZone($timezone)
                 );
-                $date = $formatter->parse($date) ?: (new \DateTime($date))->getTimestamp();
+
+                $localeDate = null;
+                if (\Zend_Locale::isLocale($locale)) {
+                    $localeDate = $locale;
+                }
+                $date = $formatter->parse($date) ?: (new \Zend_Date(
```

Comment:
`Zend_Locale` and `Zend_Date` must not be used, there was a separate refactoring effort to get rid of them and some static tests are checking they are not used.

There should be some `intl` or PHP library equivalents.

Reply:
@orlangur I have used this because the format date is in Zend. I'm going to review it again

Reply:
@orlangur What can you recommend me to continue with this?

Reply:
Only native `intl` classes, like `\DateTime`, `\Locale` etc.